src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Controllers/DraftApprenticeshipController.cs

```diff
@@ -279,6 +279,42 @@ public async Task<IActionResult> AddDraftApprenticeship(string changeCourse, str
             }
         }
 
+        [HttpGet]
+        [Route("add/apprenticeship/overlap-alert", Name = RouteNames.DraftApprenticeshipOverlapAlert)]
+        public IActionResult DraftApprenticeshipOverlapAlert(DraftApprenticeshipOverlapAlertRequest request, [FromServices] IFeatureTogglesService<ProviderFeatureToggle> featureTogglesService)
+        {
+            var featureToggleEnabled = featureTogglesService.GetFeatureToggle(ProviderFeature.OverlappingTrainingDate).IsEnabled;
+
+            DraftApprenticeshipViewModel model;
+            if (request.DraftApprenticeshipHashedId != null)
+            {
+                model = PeekStoredEditDraftApprenticeshipState();
+            }
+            else
+            {
+                model = PeekStoredAddDraftApprenticeshipState();
+            }
+
+            var vm = new DraftApprenticeshipOverlapAlertViewModel
+            {
+                OverlappingTrainingDateRequestToggleEnabled = featureToggleEnabled,
```

Comment:
this property of the model isn't used and can be removed. As can the variable above and the injected feature toggle service

Reply:
updated